docs/Researcher/cli-reference/runai-submit-dist-xgboost.md

```diff
@@ -0,0 +1,352 @@
+## Description
+
+Submit a distributed XGBoost training run:ai job to run.
+
+Syntax notes:
+
+* Options with a value type of *stringArray* mean that you can add multiple values. You can either separate values with a comma or add the flag twice.
+
+## Examples
+
+```console
+runai submit-dist xgboost --name distributed-job --workers=2 -g 1 \
+	-i <image_name
+>
+```
+
+## Options
+
+### Distributed
+
+#### --clean-pod-policy < string >
+
+> The CleanPodPolicy controls deletion of pods when a job terminates. The policy can be one of the following values:
+>
+>* **Running**&mdash;only pods still running when a job completes (for example, parameter servers) will be deleted immediately. Completed pods will not be deleted so that the logs will be preserved. (Default)
+>* **All**&mdash;all (including completed) pods will be deleted immediately when the job finishes.
+>* **None**&mdash;no pods will be deleted when the job completes.
+
+#### --non-preemptible
+
+> Resources for non-preemptible jobs are guaranteed and will not be reclaimed at any time
```

Comment:
@itayvallach we agreed to take this out, in the code & the documentation @jasonnovichRunAI